contracts/bridge/contracts/LightClientBridge.sol

```diff
@@ -198,194 +262,187 @@ contract LightClientBridge is Pausable, Initializable, ValidatorRegistry {
 
         // Accept and save the commitment
         validationData[currentId] = ValidationData(
+            msg.sender,
             commitmentHash,
-            validatorClaimsBitfield,
-            block.number
+            block.number,
+            validatorClaimsBitfield
         );
 
         emit InitialVerificationSuccessful(msg.sender, block.number, currentId);
 
-        currentId = currentId.add(1);
+        currentId = currentId + 1;
```

Comment:
suggest: return the currentId, and relayer know which id to complete the two-step verification.

Reply:
Done

Reply:
note: the returned `currentId-1` is the initialTx ID